### DISCUSSION TITLE
How to get last attemted proof info using cli command?

### DISCUSSION THREAD

**Discussion by losingle** [Chia-Network/chia-blockchain#5994] [Support]
How to get last attemted proof info using cli command?

ubuntu 20.04
chia 1.1.6

**Comment by BrandtH22** [comment]
@losingle , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - you will need to review the log file and have it set to INFO level to get that information 👍 

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
